### PR TITLE
Fix/reverting spring boot2 spring5

### PR DESCRIPTION
**Pull request by whoswendy**
This PR contains code to revert to springboot2 and spring5.

Changes include:
- dependency version changes
- jakarta -> javax
- httpStatusCode -> httpStatus

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
 
         <!--Logging-->
         <log4j.version>2.19.0</log4j.version>
-        <slf4j.version>2.0.4</slf4j.version>
+        <sl4j.version>1.7.36</sl4j.version>
```
Comment (eaquin02) [inline pom.xml]: Why did slf4j change to sl4j?

File: service/synapse-service-graphql/pom.xml [hunk 2]
```diff
@@ -38,17 +38,21 @@
...
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>
+            <groupId>com.graphql-java-kickstart</groupId>
+            <artifactId>graphiql-spring-boot-starter</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.springframework.security</groupId>
-            <artifactId>spring-security-web</artifactId>
+            <groupId>com.graphql-java-kickstart</groupId>
+            <artifactId>voyager-spring-boot-starter</artifactId>
```
Comment (eaquin02) [inline service/synapse-service-graphql/pom.xml:55]: whats this change for?

File: service/synapse-service-graphql/pom.xml [hunk 3]
```diff
@@ -38,17 +38,21 @@
         </dependency>
 
         <!--GraphQL-->
+        <dependency>
+            <groupId>com.graphql-java</groupId>
+            <artifactId>graphql-java-extended-scalars</artifactId>
+        </dependency>
```
Comment (andCastStatic) [inline service/synapse-service-graphql/pom.xml:44]: Is there a reason why this is added?

File: service/synapse-service-graphql/pom.xml [hunk 4]
```diff
@@ -38,17 +38,21 @@
...
         <dependency>
             <groupId>com.graphql-java-kickstart</groupId>
             <artifactId>graphql-spring-boot-starter</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>
+            <groupId>com.graphql-java-kickstart</groupId>
+            <artifactId>graphiql-spring-boot-starter</artifactId>
```
Comment (andCastStatic) [inline service/synapse-service-graphql/pom.xml:51]: What is this change for?

File: pom.xml (hunk 1 above)
Comment (whoswendy) [reply pom.xml]: it was a typo, fixed it

File: service/synapse-service-graphql/pom.xml (hunk 2 above)
Comment (whoswendy) [reply service/synapse-service-graphql/pom.xml:55]: This is needed for graphql to work, reverting the changes that was done in previous pr when upgrading to springboot3

File: service/synapse-service-graphql/pom.xml (hunk 3 above)
Comment (whoswendy) [reply service/synapse-service-graphql/pom.xml:44]: This is needed for graphql to work, reverting the changes that was done in previous pr when upgrading to springboot3

File: service/synapse-service-graphql/pom.xml (hunk 4 above)
Comment (whoswendy) [reply service/synapse-service-graphql/pom.xml:51]: This is needed for graphql to work, reverting the changes that was done in previous pr when upgrading to springboot3

**Review by andCastStatic — APPROVED** [review]

**Pull request closed without merging**